pkg/scheduler/core/generic_scheduler.go

```diff
@@ -1053,9 +1058,13 @@ func filterPodsWithPDBViolation(pods []interface{}, pdbs []*policy.PodDisruption
 					continue
 				}
 				// We have found a matching PDB.
-				if pdb.Status.PodDisruptionsAllowed <= 0 {
+				// All pods matching same PDB are cotrolled by only one controller
+				// then they have same priprities.
```

Comment:
I don't think the added comment is useful, I suggest removing it.

Reply:
Done.